distributed/worker.py

```diff
@@ -2592,9 +2594,25 @@ def _handle_instructions(self, instructions: list[Instruction]) -> None:
         for inst in instructions:
             if isinstance(inst, SendMessageToScheduler):
                 self.batched_stream.send(inst.to_dict())
+            elif isinstance(inst, Execute):
+                self.loop.add_callback(
+                    self._async_instruction_callback,
+                    self.execute(inst.key, stimulus_id=inst.stimulus_id),
+                    stimulus_id=inst.stimulus_id,
+                )
```

Comment:
This differs from the design document, which instead was creating a fire-and-forget task.
As far as I understand, the difference between the two is purely cosmetic.

Added value would be given from spawning a task, track it e.g. in a set ``Worker.running_asyncio_tasks``, and then cancel it in ``Worker.close()``. Even if desirable, however, I think this is best left to a future PR.